airflow/operators/python.py

```diff
@@ -403,6 +399,25 @@ def get_python_source(self):
         """Return the source of self.python_callable."""
         return textwrap.dedent(inspect.getsource(self.python_callable))
 
+    def _validate_python_callable(self, python_callable):
```

Comment:
I initially had an idea of suggesting a common validation method that is invoked for any kind of operator. This can help us to place all the pre-requisite checks w.r.t any operator. Probably enforce it to be implemented in the inherited classes by making it an abstract method. But, for now, I've wrote this as I don't understand the codebase enough.